packages/dds/tree/src/test/tableSchema.spec.ts

```diff
@@ -86,7 +87,7 @@ describe("TableFactory unit tests", () => {
 
 	describe("Column Schema", () => {
```

Comment:
you could have at least one test which creates columns who's schema don't match the table (subset, superset and intersection cases probably).

Currently its unclear if the subset case is supposed to be allowed, but in another comment I suggested suported it. Regardless of if its supported, it should be tested to either work or error

Reply:
I'm going to do this in a future PR, I think. But I agree that this should be done.